virttest/qemu_vm.py

```diff
@@ -4784,6 +4785,10 @@ def process_info_block(self, blocks_info):
         """
         block_list = []
         block_entry = []
+
+        if not isinstance(blocks_info,str):
+            return block_list
```

Comment:
I am a bit confused, which the situation that `blocks_info` will be passed by `list` object?

Reply:
`       blocks_info = self.monitor.info("block")
        block = self.get_block_old(blocks_info, p_dict)
        if block:
            return block

        block_list = self.process_info_block(blocks_info)`

get_block_old failed will call process_info_block